indico/modules/events/abstracts/fields.py

```diff
@@ -118,9 +118,6 @@ def __init__(self, *args, **kwargs):
         self.empty_message = _('There are no authors')
         super().__init__(*args, **kwargs)
 
-    def _convert_data(self, data):
-        return list({self._get_person_link(x) for x in data})
```

Comment:
the deduplication (that's why it's converting to a set and then back to a list) is no longer needed?

Reply:
Yes, it is needed, I ignored the de-duplication here while trying to avoid repeating it, thanks!